git/content/git.tex

```diff
@@ -130,6 +130,22 @@
   \end{center}
 \end{frame}
 
+\begin{frame}{Konzept: Commits}
+  \begin{itemize}
+    \item Stand des Repositories zu einem Zeitpunkt
+    \item Erlaubt das Hinzufügen von Kommentaren: Was wurde getan seit dem letzten Commit?
+    \item Platzsparend: Nur die Änderungen zum letzten Commit werden
```

Comment:
Auch das ist nicht richtig.

Reply:
Knappe Zusammenfassung der details: https://github.com/kdakan/How-Git-Works

Reply:
Was ist daran falsch? Nur veränderte Dateien werden eingepflegt. Wenn sich der Hash nicht ändert wird kein neuer "Blob" in Gut eingepflegt, die alte Datei wird im neuen Commit weiter über ihren Hash referenziert.

Reply:
Es werden aber nicht die Änderungen gespeichert sondern die ganzen geänderten Dateien. Eventuell aber noch "delta komprimiert". Es ist also nicht so einfach und es entsteht schnell das Bild git würde nur die "diffs" speichern, was nicht korrekt ist.